### PR TITLE
Fix OOM errors in `batchnorm` and `instancenorm`

### PR DESCRIPTION
**Pull request by Priya2698**
- Uses `torch.cuda.empty_cache()` to avoid OOM errors when running a large number of benchmarks.
- Reduce maximum size of 4D tensor to run on A100 80GB.

### CODE REVIEW

**Comment by Priya2698** [comment]
@xwang233 Is there a way to run this through CI to see if we get any segmentation errors?

**Review by naoyam — APPROVED** [review]
LGTM

**Comment by xwang233** [comment]
I've started a pipeline to test this build and python benchmarks https://nv/pytorch-ci/-/pipelines/10454231

**Comment by xwang233** [comment]
The CI for python benchmarks for this PR branch looks good! Thanks for the fix.

**Review by xwang233 — APPROVED** [review]
